Street Trees/STConfirmationPageViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  STConfirmationPageViewController.swift
+//  Street Trees
+//
+//  Created by Pedro Trujillo on 10/17/16.
+//  Copyright © 2016 Code for Orlando. All rights reserved.
+//
+
+import UIKit
+import StreetTreesTransportKit
+
+
+//**********************************************************************************************************************
+// MARK: - Protocol
+
+protocol STConfirmationPageViewControllerDelegate: NSObjectProtocol {
+    func ConfirmationFormViewController(form: STConfirmationPageViewController, didCompleteWithWufooForm anWufooForm: STTKWufooForm)
```

Comment:
Change to: 
func confirmationFormViewController(form: STConfirmationPageViewController, didCompleteWithWufooForm aWufooForm: STTKWufooForm)